### PR TITLE
Main layout

### PR DESCRIPTION
**Pull request by jeetmitra01**

### CODE REVIEW

File: client/src/components/Layout.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import "./App.css";
+
+function Layout(props: { children: typeof React.Component, navbar: typeof React.Component }) {
```
Comment (sselkef) [inline client/src/components/Layout.tsx:4]: navbar is required to be a React.Component here but is a javascript element in the project.
Not sure if this is correct, but it seems to be the issue when trying to include the Layout component in the main app.

**Review by absmith22 — COMMENTED** [review]
I cannot seem to view any content when running the client. The layout code looks correct but I am unsure if I should be seeing anything.

**Review by absmith22 — DISMISSED** [review]
All set after many rounds of waiting for the client to start!

**Review by absmith22 — APPROVED** [review]
Fixed Lint and Client Test errors

**Comment by sselkef** [comment]
Should LoginForm also be imported into App.tsx? Or would it stay as a subcomponent/prop for other components in the future?
Otherwise, everything else seems fine.

**Comment by benmelz** [comment]
> Should LoginForm also be imported into App.tsx? Or would it stay as a subcomponent/prop for other components in the future?
> 
> Otherwise, everything else seems fine. 

Yeah, I see the argument that it's only gonna be used once, but if we do that for everything then we will have a monstrously large App.tsx and lots and lots of merge conflicts.

**Comment by benmelz** [comment]
> Should LoginForm also be imported into App.tsx? Or would it stay as a subcomponent/prop for other components in the future?
> 
> Otherwise, everything else seems fine. 

That's being taken care of by @daniel-melanson and @Fish7793 in their integration PR

**Review by sselkef — APPROVED** [review]
Apologies for the late approval.

**Pull request merged**